processes/anomaly_detection.json

```diff
@@ -71,7 +71,7 @@
     "returns": {
         "description": "A file with the CropSAR fAPAR values of the individual fields and the regional average fAPAR value for the specified time period",
         "schema": {
-            "type": "data cube"
+            "type": "DataCube"
```

Comment:
type: object
subtype: raster-cube

Reply:
thank you. Updated

Reply:
Hmm... the return values don't make sense to me as they are. The description of burntMapping_chunks says it returns a GeoTiff, but the schema says it returns a raster data cube. The raster data cube seems correct due to the last node being merge_cubes. So this needs a description that describes how the data cube looks like. What dimensions are returned? What about the values? ...